README.md

```diff
@@ -2,147 +2,142 @@
 
 [![](https://dcbadge.vercel.app/api/server/cudamode?style=flat)](https://discord.gg/cudamode)
 
-This repository is currently under heavy development - if you have suggestions on the API or use-cases you'd like to be covered, please open an [issue](https://github.com/pytorch/ao/issues)
 
 ## Introduction
-`torchao` is a PyTorch library for quantization and sparsity.
+
+torchao is a library which makes it easy to integrate and create high performance kernels with custom data types and layouts with up to
```

Comment:
create, integrate and compare?

Reply:
Not sure how I feel about compare being in the first line, if the intent is to say carefully benchmarked then I feel that's table stakes for a library about performance and not worth explicitly mentioning

Reply:
when I read this sentence, to me torchao comes across as a kernel library with custom types. But I think what we want to say is something like "torchao is a library to create and integrate high-performance custom data types, layouts and kernels into their PyTorch workflows with up to.." wdyt?

Reply:
Yeah that's much clearer